docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  mintr:
+    image: mrcide/mintr:${GIT_BRANCH}
```

Comment:
probably this should be GIT_SHA (or whatever we call it) because that'll be the only version reliably present normally, and using this means you won't get hard-to-debug test failures when the agent does not pull the image (I think it will pull the first time a tag is not found)